src/SonarQube.Client/Api/Requests/V7_20/GetIssuesRequest.cs

```diff
@@ -97,9 +73,9 @@ private class ServerIssue
             [JsonProperty("line")]
             public int? Line { get; set; }
             [JsonProperty("message")]
-            public string Message { get; set; }            
-            [JsonProperty("resolution")]
-            public string Resolution { get; set; }            
```

Comment:
We don't care about the resolution state but only about the status for our work.